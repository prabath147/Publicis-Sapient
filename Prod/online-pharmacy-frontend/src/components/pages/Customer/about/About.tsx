import { Container } from '@mantine/core';
import React from 'react';

export default function About() {
  return (
    <Container>
      <div data-testid="about">About</div>
    </Container>
  );
}
