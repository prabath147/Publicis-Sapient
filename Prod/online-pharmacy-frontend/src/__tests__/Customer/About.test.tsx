import React from 'react';
import renderer from 'react-test-renderer';
import About from '../../components/pages/Customer/about/About';

it('component created', () => {
    const component = renderer.create(
        <About />
    )
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})