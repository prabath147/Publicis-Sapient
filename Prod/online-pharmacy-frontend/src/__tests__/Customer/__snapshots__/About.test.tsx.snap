// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component created 1`] = `
<div
  className="mantine-Container-root mantine-15pjuqq"
>
  <div
    data-testid="about"
  >
    About
  </div>
</div>
`;
